app/controllers/rx_controller.rb

```diff
@@ -4,13 +4,25 @@
 class RxController < ApplicationController
   include ActionController::Serialization
 
-  skip_before_action :authenticate
+  before_action :authorize_rx
   before_action :authenticate_client
 
   protected
 
   def client
-    @client ||= Rx::Client.new(session: { user_id: ENV['MHV_USER_ID'] })
+    @client ||= Rx::Client.new(session: { user_id: mhv_correlation_id })
+  end
+
+  def authorize_rx
+    mhv_correlation_id || raise_access_denied
```

Comment:
I mentioned this in @austin-martinez but not sure what became of it - mhv_id being `nil` could be as a result of 1 of three cases:

1. The user has not authenticated at all yet (which would get picked up by the before_action :authenticate)
2. The User is LOA 1
3. The user is LOA 3 but MVI failed to getch mhv_id somehow.

It's my understanding that the front-end will try and recover from **case 2** by up-leving the users' session from 1 to 3. If you return 403 for **case 3**, it could result in an endless loop where the front end keeps telling the user to up-level when really that is not the problem.